src/compiler/diagnosticMessages.json

```diff
@@ -859,6 +859,14 @@
         "category": "Error",
         "code": 1319
     },
+    "String literal with double quotes expected.": {

```

Comment:
This is not just for the property name though. It could be value that is string literal. Json file can only accept string literals that are double quoted